### DISCUSSION TITLE
Manage SSH keys

### DISCUSSION THREAD

**Discussion by Sigafoos** [GitJournal/GitJournal#718] [Feature Requests]
## Environment

**App version:** 1.73.3+2106
**Device information:**
 - OS name and version: iOS 14.2
 - Manufacturer: Apple
 - Model: iPhone XS Max

## Description

**What you'd like to happen:** An interface to view, add and delete ssh keys when not adding a repo. I tried adding the host and there was a permission error: I may not have copied the key to my Gitea settings properly. And then there was no way to see the public key, or to try again with the same settings. I had to start again.

**Alternatives you've considered:**   Generate a keypair, save it to my files, and import that way.

**Images:**

**Comment by vHanda** [comment]
Once I finish multi-repo support, this might be nice to do, as I'll also be exposing the concept of multiple remotes.

**Comment by eichin** [comment]
(android 1.83.10) I have a simpler use case in mind - I have a new custom git repo URI but I want to use the same key (that gitjournal previously generated) with it.  I don't actually need to see it, just reuse it.  (I apparently couldn't use the "File" buttons to do that, though I did poke around a bit.)
